tests/unittest/_torch/visual_gen/test_visual_gen_args.py

```
@@ -210,8 +210,8 @@ def test_quant_config_dict_passthrough(self):
         assert isinstance(args.quant_config, dict)
         assert args.quant_config["quant_algo"] == "FP8"
         # The same dict is the source of truth for the derived flags; verify
-        # the parser DiffusionModelConfig.from_pretrained will run on it.
-        qc, _, dwq, daq = DiffusionModelConfig.load_diffusion_quant_config(args.quant_config)
+        # the pipeline-config parser will run on it.
+        qc, _, dwq, daq = DiffusionPipelineConfig.load_diffusion_quant_config(args.quant_config)
```

Ruff (0.15.15)

[warning] Unpacked variable `daq` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)